src/components/PoolList/ListItem.tsx

```diff
@@ -275,6 +275,8 @@ const ListItem = ({ pool, subgraphPoolData, myLiquidity, oddRow }: ListItemProps
   const fee = subgraphPoolData.oneDayFeeUSD ? subgraphPoolData.oneDayFeeUSD : subgraphPoolData.oneDayFeeUntracked
 
   const oneYearFL = getOneYearFL(subgraphPoolData.reserveUSD, fee).toFixed(2)
+  
+  const ampLiquidity = formattedNum(`${(+amp.toSignificant(5)) * (+subgraphPoolData.reserveUSD)}`, true)
```

Comment:
Using "+" to convert string to number is unsafe

You can try on the browser console:

`+'2,000'` => NaN
`parseFloat('2,000')` => 2

So we should use parseFloat

Reply:
updated